android-exoplayer/src/main/java/com/brentvatne/exoplayer/ReactExoplayerView.java

```diff
@@ -773,11 +774,54 @@ private void releasePlayer() {
         bandwidthMeter.removeEventListener(this);
     }
 
+       private static class OnAudioFocusChangedListener implements AudioManager.OnAudioFocusChangeListener {
+            private final ReactExoplayerView view;
```

Comment:
We'd better define the `view` as a weak reference. The static class keeps referencing the `view`, it will prevent the `view` from being released. 
e.g. `WeakReference<ReactExoplayerView> view`

Reply:
good idea, I will check later if it can actually help to fix the memory leak

Reply:
it slightly larger change though, so I will do it the next PR

Reply:
I would avoid relying on weak references as a solution. One problem is that you could find the view is mistakenly garbage collected earlier than it should be. 

Instead we should make sure we correctly remove references when the player is unloaded.